contracts/VaultV3.vy

```diff
@@ -1338,6 +1383,27 @@ def _process_report(strategy: address) -> (uint256, uint256):
     return (gain, loss)
 
 # SETTERS #
+
+@external
+def setName(name: String[64]):
+    """
+    @notice Change the vault name.
+    @dev Can only be called by the Role Manager.
+    @param name The new name for the vault.
+    """
+    assert msg.sender == self.role_manager, "not allowed"
+    self.name = name
```

Comment:
event seems unnecessary since the name is not expected to be updated often or needed to be stored locally. Understanding the current state is also not dependent on knowing when the name was updated.